control-plane-operator/controllers/hostedcontrolplane/assets/olm/catalog-operator-deployment.yaml

```diff
@@ -34,7 +34,24 @@ spec:
         - name: kubeconfig
           secret:
             secretName: service-network-admin-kubeconfig
+        - name: oas-konnectivity-proxy-cert
+          secret:
+            secretName: konnectivity-client
       containers:
+        - name: socks5-proxy
+          image: {{ imageFor "hosted-cluster-config-operator" }}
```

Comment:
You probably need a `command` here as well, since the image entrypoint is not socks proxy binary.
and args as well?

Reply:
This works because we default to expect values but adding the command / args can't hurt.

Reply:
Sorry - most of the testing was done with a different dockerfile, this won't work right now. I'll add the args and cmd.

Reply:
Ended up just adding the command as the flags default to the correct values.